packages/babel-generator/test/index.js

```diff
@@ -367,7 +368,7 @@ suites.forEach(function(testSuite) {
             if (actualCode) {
               const actualAst = parse(actualCode, {
                 filename: actual.loc,
-                plugins: [
+                plugins: task.options.plugins || [
```

Comment:
maybe it's better we be explicit in the tests 👍 , could probably remove flow (or all of them) and add a plugin option for each test later.